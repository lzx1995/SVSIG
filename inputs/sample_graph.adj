AdjacencyGraph
22
20
0
0
1
3
4
4
5
7
8
10
10
11
12
13
14
15
15
17
18
19
20
20
2
3
4
4
4
5
7
8
2
5
12
20
19
17
21
14
14
16
20
12
