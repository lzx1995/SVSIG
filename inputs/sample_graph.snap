1	2
2	3
2	4
3	4
5	4
6	7
6	5
7	8
8	5
8	2
10	12
11	20
12	19
19	12
18	20
14	21
13	17
16	14
16	14
17	16